libs/application/template-api-modules/README.md

```diff
@@ -35,7 +35,7 @@ export class ReferenceTemplateModule {
   static register(config: BaseTemplateAPIModuleConfig): DynamicModule {
```

Comment:
We should use the opportunity to refactor these modules from DynamicModule and remove the register function as it is no longer needed.

Reply:
done